src/Symfony/Component/Serializer/Annotation/Groups.php

```diff
@@ -17,12 +17,14 @@
  * Annotation class for @Groups().
  *
  * @Annotation
+ *
```

Comment:
let's remove the added empty lines, we don't agree with fabbot on these

Reply:
@nicolas-grekas done https://github.com/symfony/symfony/pull/49594/commits/632c8ea722a6239e5e9ca7ef52f0a7d1ba2cdcb1